### PR TITLE
ai slop

### PR DESCRIPTION
**Pull request by robobun**
This PR has been marked as AI slop and the description has been updated to avoid confusion or misleading reviewers.

Many AI PRs are fine, but sometimes they submit a PR too early, fail to test if the problem is real, fail to reproduce the problem, or fail to test that the problem is fixed. If you think this PR is not AI slop, please leave a comment.

### CODE REVIEW

File: test/cli/init/init.test.ts [hunk 1]
```diff
@@ -207,6 +212,7 @@ import path from "path";
     expect(await Bun.file(path.join(temp, "mydir/.gitignore")).text()).toMatchInlineSnapshot(`"my edited .gitignore"`);
     expect(await Bun.file(path.join(temp, "mydir/package.json")).json()).toMatchInlineSnapshot(`
     {
+      "$schema": "https://json.schemastore.org/package.json",
```
[the remark this answers is not in the document]
Comment (robobun) [reply test/cli/init/init.test.ts]: Added explicit `not.toHaveProperty("$schema")` assertion in the "bun init twice" test. The test now writes a package.json without `$schema` and verifies init doesn't inject it.

File: src/cli/init/tsconfig.default.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
```
[the remark this answers is not in the document]
Comment (robobun) [reply src/cli/init/tsconfig.default.json:2]: Fixed in 735edeb. Added a separate `jsconfig.default.json` template with `https://json.schemastore.org/jsconfig` and updated the init code to select the correct asset based on the entry point loader.

File: src/cli/init_command.zig [hunk 3]
```diff
@@ -814,11 +816,11 @@ pub const InitCommand = struct {
...
-                        const filename = if (loader.isTypeScript())
-                            "tsconfig.json"
-                        else
-                            "jsconfig.json";
-                        Assets.createFull("tsconfig.json", filename, " (for editor autocomplete)", false, .{}) catch break :brk;
+                        if (loader.isTypeScript()) {
+                            Assets.createFull("tsconfig.json", "tsconfig.json", " (for editor autocomplete)", false, .{}) catch break :brk;
+                        } else {
+                            Assets.createFull("jsconfig.json", "jsconfig.json", " (for editor autocomplete)", false, .{}) catch break :brk;
+                        }
```
[the remark this answers is not in the document]
Comment (robobun) [reply src/cli/init_command.zig:824]: The non-index JS edge case (empty entry_point defaulting to TS loader) is pre-existing behavior unrelated to this PR. This PR does not change the loader detection logic. Filed as a separate concern if needed.

File: test/cli/init/init.test.ts [hunk 4]
```diff
@@ -38,6 +39,9 @@ import path from "path";
     expect(fs.existsSync(path.join(temp, ".gitignore"))).toBe(true);
     expect(fs.existsSync(path.join(temp, "node_modules"))).toBe(true);
     expect(fs.existsSync(path.join(temp, "tsconfig.json"))).toBe(true);
+
+    const tsconfig = fs.readFileSync(path.join(temp, "tsconfig.json"), "utf8");
+    expect(tsconfig).toContain('"$schema": "https://json.schemastore.org/tsconfig"');
   }, 30_000);
 
   test("bun init with piped cli", async () => {
```
[the remark this answers is not in the document]
Comment (robobun) [reply test/cli/init/init.test.ts:47]: Added test "bun init with js entry point creates jsconfig.json" in 7079c92 — creates a dir with index.js, asserts jsconfig.json exists (not tsconfig.json), and checks for the correct jsconfig schema URL.
